docs/getting_started.rst

```diff
@@ -511,72 +512,38 @@ working Qt installation. We can install these archives using this command:
 Installing Modules With Archives Specified
 ``````````````````````````````````````````
 
-Now let's say we need to install the ``qtcharts`` and ``qtlottie`` modules.
-Let's see what archives are part of these modules:
+As of aqt v2.1.0, the ``--archives`` flag will only apply to
```

Comment:
I’m just guessing at what version this would be part of; please let me know if I should change “v2.1” to something else.